tests/lax_numpy_test.py

```diff
@@ -1754,6 +1754,16 @@ def testIntegerPower(self, ptype):
     self.assertLen(eqns, 1)
     self.assertEqual(eqns[0].primitive, lax.integer_pow_p)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_x={}_y={}".format(x, y), "x": x, "y": y}
+      for x in [-1, 0, 1]
```

Comment:
It would not be an error for JAX to produce anything at all if the value overflows. So this test may be too strict. Maybe add a comment to that effect?

Reply:
Actually, never mind, I see that the values of `x` can't cause overflows.